client/src/shared/constants.tsx

```diff
@@ -96,6 +96,8 @@ export const TASK_DISPATCHER_NAMES = [
     'subflow',
 ];
 
+export const ROOT_CLUSTER_ELEMENT_NAMES = ['aiAgent'];
```

Comment:
@ivicac ode nam triba točan popis componentName-ova osim aiAgenta koji su root cluster elementi